### ISSUE TITLE
Got Error: Command failed with exit code 1: mv *.* form

### ISSUE THREAD

**Issue by andreasnrb** [dgwyer/create-wp-block#14]
Ran the following command:
"npx create-wp-block test-forms -b form text-field button -tw"

Got following error after process was complete. There were no blocks in the created folder. There was an empty folder named "form" in the src folder.

Following files were in src folder.
- block.json
- edit.js
- editor.scss
- form/
- index.js
- save.js
- style.scss

**The error:**
```
Post processing...

file:///home/andreas/.npm/_npx/bab405a38ec78caa/node_modules/execa/lib/error.js:59
                error = new Error(message);

Error: Command failed with exit code 1: mv *.* form
mv: cannot stat '*.*': No such file or directory
    at makeError (file:///home/andreas/.npm/_npx/bab405a38ec78caa/node_modules/execa/lib/error.js:59:11)
    at execaSync (file:///home/andreas/.npm/_npx/bab405a38ec78caa/node_modules/execa/index.js:193:17)
    at execaCommandSync (file:///home/andreas/.npm/_npx/bab405a38ec78caa/node_modules/execa/index.js:234:9)
    at file:///home/andreas/.npm/_npx/bab405a38ec78caa/node_modules/create-wp-block/index.js:92:13
    at Array.forEach (<anonymous>)
    at file:///home/andreas/.npm/_npx/bab405a38ec78caa/node_modules/create-wp-block/index.js:85:12
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5) {
  shortMessage: 'Command failed with exit code 1: mv *.* form',
  command: 'mv *.* form',
  escapedCommand: 'mv "*.*" form',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: "mv: cannot stat '*.*': No such file or directory",
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

**Comment by khottavong-jg** [comment]
I'm also seeing this issue

**Comment by dgwyer** [comment]
Thanks for reporting the issue. I just ran the same command:

```
npx create-wp-block test-forms -b form text-field button -tw
```

But didn't have any errors:

![explorer](https://user-images.githubusercontent.com/1482075/170738066-95e6b7b3-2071-4225-b69d-6129e0f232fb.png)

![terminal](https://user-images.githubusercontent.com/1482075/170738090-a63a803b-ff78-4095-a87a-b26de04ec4e2.png)

I'm on Windows and used Git Bash to run the command. I don't have time to look into possible causes right now but please feel free to submit a PR or leave feedback if you have any ideas as to why this may be happening for you. Maybe it's down to macOS? I've only tested on Windows so far.

**Comment by khottavong-jg** [comment]
Ahh I'm on macOS so that's most likely the issue

**Comment by dgwyer** [comment]
It 'should' work on either OS but I don't have a setup to debug on macOS right now.

**Comment by andreasnrb** [comment]
I was running the command in WSL2 and Ubuntu.

Den fre 27 maj 2022 kl 18:29 skrev David Gwyer ***@***.***>:

**Comment by mdjwel** [comment]
I'm also facing the same issue on Mac OS. Probably, there is something misconfigured for Mac OS. Could you try it on Mac OS, please?

**Comment by tyte** [comment]
I experience same issue on Mac OS, though I didn't use "-tw".

**Comment by dgwyer** [comment]
I'll hopefully be taking a look at this issue soon.

**Comment by dgwyer** [comment]
@andreasnrb @tyte @mdjwel This should be fixed in v2.7.0. I've tested it on macOS and it works fine now. If you can please try it out and let me know if there are any issues.

https://www.npmjs.com/package/create-wp-block

**Comment by andreasnrb** [comment]
Still have same error when using Ubuntu and WSL2 using version:2.7.0. Node v19 was used. NPX was 8.19.2.

Both with --tw and without I get the below error when running  **npx create-wp-block test-forms -b form text-field button**
I get **no** error when I run **npx create-wp-block test-forms -b form**
Issue seems to be related to the multiple blocks part.

```
Post processing...

Installing blocks: form text-field button
file:///home/andreas/.nvm/versions/node/v19.0.0/lib/node_modules/create-wp-block/node_modules/execa/lib/error.js:59
                error = new Error(message);
                        ^

Error: Command failed with exit code 1: mv test-forms/src/tailwind.config.js test-forms/src/form
mv: cannot stat 'test-forms/src/tailwind.config.js': No such file or directory
    at makeError (file:///home/andreas/.nvm/versions/node/v19.0.0/lib/node_modules/create-wp-block/node_modules/execa/lib/error.js:59:11)
    at execaSync (file:///home/andreas/.nvm/versions/node/v19.0.0/lib/node_modules/create-wp-block/node_modules/execa/index.js:193:17)
    at execaCommandSync (file:///home/andreas/.nvm/versions/node/v19.0.0/lib/node_modules/create-wp-block/node_modules/execa/index.js:234:9)
    at file:///home/andreas/.nvm/versions/node/v19.0.0/lib/node_modules/create-wp-block/index.js:120:9
    at Array.forEach (<anonymous>)
    at file:///home/andreas/.nvm/versions/node/v19.0.0/lib/node_modules/create-wp-block/index.js:109:12
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  shortMessage: 'Command failed with exit code 1: mv test-forms/src/tailwind.config.js test-forms/src/form',
  command: 'mv test-forms/src/tailwind.config.js test-forms/src/form',
  escapedCommand: 'mv "test-forms/src/tailwind.config.js" "test-forms/src/form"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: "mv: cannot stat 'test-forms/src/tailwind.config.js': No such file or directory",
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}

```
